### PR TITLE
Multiple devtools plugin fix

### PR DESCRIPTION
**Pull request by mercillo**
### Change Type (required)
Indicate the type of change your pull request is:

- [ ] `patch`
- [x] `minor`
- [ ] `major`
dset also contained a merge that allowed multiple plugins to be run at the same time. Using lodash.set was causing it to overwrite the existing plugin data. Testing canary to see if this is the right issue, then will add tests to account for dset error when throwing null.

<details>
  <summary>📦 Published PR as canary version: <code>0.8.2--canary.174.3739</code></summary>
  <br />

  Try this version out locally by upgrading relevant packages to 0.8.2--canary.174.3739
</details>

### CODE REVIEW

File: devtools/plugins/desktop/test-env/setup.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ VERDACCIO_REGISTRY="http://localhost:4873"
 PLUGINS=("@player-tools/devtools-basic-web-plugin" "@player-tools/devtools-profiler-web-plugin")
 
 # Define browser-devtools depenedencies
-BROWSER_DEVTOOLS_DEPS=("@player-ui/pubsub-plugin" "dequal" "@player-tools/devtools-client@0.0.0-PLACEHOLDER" "@player-tools/devtools-messenger@0.0.0-PLACEHOLDER" "@player-tools/devtools-types@0.0.0-PLACEHOLDER")
+BROWSER_DEVTOOLS_DEPS=("@player-ui/pubsub-plugin" "dequal" "@player-tools/devtools-client@0.8.1" "@player-tools/devtools-messenger@0.8.1" "@player-tools/devtools-types@0.8.1")
```
Comment (rafbcampos) [inline devtools/plugins/desktop/test-env/setup.sh:21]: wouldn't have an issue testing local changes setting the current versions here, instead of the versions we'd get publishing to verdaccio?

File: devtools/plugins/desktop/basic/src/WrapperComponent.tsx [hunk 2]
```diff
@@ -156,7 +157,12 @@ export const WrapperComponent = ({
     if (dequal(state.plugins[pluginID]?.flow?.data?.data, data)) return;
 
     const newState = produce(state, (draft) => {
-      set(draft, ["plugins", pluginID, "flow", "data", "data"], data);
+      try {
+        dset(draft, ["plugins", pluginID, "flow", "data", "data"], data);
+      } catch {
+        console.error("Error setting the following data: ", data);
```
Comment (rafbcampos) [inline devtools/plugins/desktop/basic/src/WrapperComponent.tsx:163]: let's create an issue to display those errors in the dev tools panel, instead of the console for easy access in the future. Maybe we can leverage the toast we created for copy to clipboard.

File: devtools/plugins/desktop/basic/src/__tests__/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  it("should initialize with default Values", () => {
+    expect(plugin.name).toBe(PLUGIN_ID);
+    expect(plugin.logs).toEqual([]);
+    expect(plugin.data).toEqual({});
+    expect(plugin.flow).toBeUndefined;
+    expect(plugin.expressionEvaluator).toBeUndefined();
+    expect(plugin.view).toBeUndefined();
+    expect(plugin.dataController).toBeUndefined();
+  });
```
Comment (rafbcampos) [inline devtools/plugins/desktop/basic/src/__tests__/index.test.tsx:57]: should we add a test for when it fails to set (e.g. try to access a property from undefined)?

**Review by rafbcampos — APPROVED** [review]
